Cargo.toml

```diff
@@ -23,16 +27,18 @@ bitvec = "1.0.1"
 clap = { version = "4.0.10", optional = true, features = ["derive"] }
 comfy-table = { version = "6.0.0", optional = true }
 config = "0.13.2"
+criterion = { version = "0.4.0", optional = true, default-features = false, features = ["async_tokio", "plotters", "html_reports"] }
```

Comment:
New? or just added explicitly so that you can limit the features?

Reply:
yea I moved it to dependencies so I can make it optional. dev-dependencies cannot be optional for some reason and we don't really need them for regular builds (just for benchmarks)